cmake/modules/AddSwiftHostLibrary.cmake

```diff
@@ -92,7 +92,7 @@ function(add_swift_syntax_library name)
   add_custom_command(
     TARGET ${target}
     POST_BUILD
-    COMMAND "${CMAKE_COMMAND}" -E touch_nocreate $<TARGET_FILE:${target}> $<TARGET_OBJECTS:${target}> "${module_base}"
```

Comment:
Is this change related?

Reply:
In an unfortunate way. The windows build fails with a fairly cryptic error, my best guess is that we're overflowing the command (either in number of arguments or length) with the extra file @rintaro added. This whole command is really just a hack to help with incremental builds, but we should check if it's needed in 3.26/3.29 and if not, I'm happy enough to just remove it.